pkg/catalog/types.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 Matrix Origin
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package catalog
+
+const (
+	// default database name for catalog
+	MO_CATALOG  = "mo_catalog"
+	MO_DATABASE = "mo_database"
+	MO_TABLES   = "mo_tables"
+	MO_COLUMNS  = "mo_columns"
+)
```

Comment:
These should not be string constants but reference to these:
https://github.com/matrixorigin/matrixone/blob/263a69919a1bd73ade7846aa2461bcd01b590c10/pkg/vm/engine/tae/catalog/model.go#L43
https://github.com/matrixorigin/matrixone/blob/263a69919a1bd73ade7846aa2461bcd01b590c10/pkg/vm/engine/tae/catalog/model.go#L45
https://github.com/matrixorigin/matrixone/blob/263a69919a1bd73ade7846aa2461bcd01b590c10/pkg/vm/engine/tae/catalog/model.go#L46
https://github.com/matrixorigin/matrixone/blob/263a69919a1bd73ade7846aa2461bcd01b590c10/pkg/vm/engine/tae/catalog/model.go#L47